### PR TITLE
include additional data fields fix #12

### LINKED ISSUE #12: Include additional fields when submitting to LeadiD
The additional Data fields are required as follows:
* loan_type
* loan_amount
* credit_rating
* property_type
* prop_address
* prop_city
* prop_state
* prop_zip
* primary_use

Please add these to the list of pipe-delimited fields in `data` that we send to LeadiD. When this is done, we'll need to coordinate with LeadiD to send a test lead.

### PR DESCRIPTION
**Pull request by jgrayson676**
I assumed that `loan_type` and `loan_amount` were mortgage specific given the other fields that were added.

Also changed travis to test node 4.1.x like you mentioned recently

### CODE REVIEW

File: spec/audit_spec.coffee [hunk 1]
```diff
@@ -54,6 +54,9 @@ describe 'Audit', ->
     it 'should have lead data in the query string', ->
       assert.match request().url, /&data=email;han@solo.com\|f_name;Han\|phone1;5125551234\|zip;78750/
 
+    it 'should have lead data of nested attributes', ->
+      assert.match request({ lead: { mortgage: { property: { postal_code: '54321' }}}}).url, /prop_zip;54321/
```
Comment (cgrayson) [inline spec/audit_spec.coffee]: Minor, but all those curly braces could be omitted.

**Comment by cgrayson** [comment]
LGTM, but please check with @alexj about which "loan type" (`loan.type` or `mortgage.loan.type`) and "loan amount" to use (or if it should look in both…?).

**Comment by jgrayson676** [comment]
Updated to use either `mortgage.loan.* fields or `loan.*` fields from discussion in #12

**Comment by cgrayson** [comment]
:+1:

**Pull request merged**